src/plugins/index_pattern_management/public/components/edit_index_pattern/edit_index_pattern.tsx

```diff
@@ -196,7 +198,99 @@ export const EditIndexPattern = withRouter(
+              id="indexPatternManagement.editIndexPattern.timeFilterLabel.timeFilterDetail"
+              defaultMessage="This page lists every field in the {indexPatternTitle} index and the field's associated core type as recorded by OpenSearch. To change a field type, use the OpenSearch"
+              values={{ indexPatternTitle: <strong>{indexPattern.title}</strong> }}
+            />{' '}
+            <EuiLink href={docLinks.links.opensearch.indexTemplates.base} target="_blank" external>
+              {mappingAPILink}
+            </EuiLink>
+          </p>
+        </EuiText>
+      );
+
+      return useUpdatedUX ? (
+        <HeaderControl
+          controls={[
+            {
+              renderComponent: component,
+            },
+          ]}
+          setMountPoint={application.setAppDescriptionControls}
+        />
+      ) : (
+        component
+      );
+    };
+
+    const renderBadges = () => {
+      if (useUpdatedUX) {
+        const components = [
+          ...(Boolean(indexPattern.timeFieldName)
+            ? [<EuiBadge color="warning">{timeFilterHeader}</EuiBadge>]
+            : []),
+          ...tags.map((tag: any) => <EuiBadge color="hollow">{tag.name}</EuiBadge>),
+        ];
+        const controls = components.map((component) => ({
+          renderComponent: component,
+        }));
+
+        return (
+          <HeaderControl controls={[...controls]} setMountPoint={application.setAppBadgeControls} />
+        );
+      } else {
+        return (
+          <EuiFlexGroup wrap>
+            {Boolean(indexPattern.timeFieldName) && (
+              <EuiFlexItem grow={false}>
+                <EuiBadge color="warning">{timeFilterHeader}</EuiBadge>
+              </EuiFlexItem>
+            )}
+            {tags.map((tag: any) => (
+              <EuiFlexItem grow={false} key={tag.key}>
+                <EuiBadge color="hollow">{tag.name}</EuiBadge>
+              </EuiFlexItem>
+            ))}
+          </EuiFlexGroup>
+        );
+      }
+    };
+
+    return useUpdatedUX ? (
+      <div data-test-subj="editIndexPattern" role="region" aria-label={headingAriaLabel}>
+        <IndexHeader
+          indexPattern={indexPattern}
+          setDefault={setDefaultPattern}
+          refreshFields={refreshFields}
+          deleteIndexPatternClick={removePattern}
+          defaultIndex={defaultIndex}
+        />
+        {showTagsSection && renderBadges()}
+        {renderDescription()}
+        {conflictedFields.length > 0 && (
+          <>
+            <EuiSpacer />
+            <EuiCallOut title={mappingConflictHeader} color="warning" iconType="alert">
+              <p>{mappingConflictLabel}</p>
+            </EuiCallOut>
+          </>
+        )}
+        <Tabs
+          indexPattern={indexPattern}
+          saveIndexPattern={data.indexPatterns.updateSavedObject.bind(data.indexPatterns)}
+          fields={fields}
+          history={history}
+          location={location}
+        />
```

Comment:
```suggestion
        const content = (
          <>
            {showTagsSection && renderBadges()}
            {renderDescription()}
            {conflictedFields.length > 0 && (
              <>
                <EuiSpacer />
                <EuiCallOut title={mappingConflictHeader} color="warning" iconType="alert">
                  <p>{mappingConflictLabel}</p>
                </EuiCallOut>
              </>
            )}
            <Tabs
              indexPattern={indexPattern}
              saveIndexPattern={data.indexPatterns.updateSavedObject.bind(data.indexPatterns)}
              fields={fields}
              history={history}
              location={location}
            />
          </>
        )
```
What about we change it like this? the content part should be the same no matter the toggle is on or off.

Reply:
Yes,  i tried to extract but fount it may be hard to maintain because there are three different spacers between two contents.

Reply:
Perhaps you can move the spacer into the `renderDescription` and `renderBadges` method. Then in that case the content will be exactly the same.

Reply:
Yes, but the spacer between Tabs and EuiCallOut maybe hard to contain.